webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -273,28 +299,35 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
-.smallTag {
-  width: 100%;
-  position: relative;
-  left: 90%;
-}
-.post-title {
-  margin-top: $space-x-small;
-  margin-bottom: $space-xx-small;
+<style lang="scss">
+.contribution-form {
+  .ds-card-image.--blur-image img {
+    filter: blur(32px);
+  }
 
-  input {
-    border: 0;
-    font-size: $font-size-x-large;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-04T19:05:42Z" title="Wednesday, December 4th 2019, 8:05:42 pm +01:00">Dec 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -183,6 +239,11 @@ export default {
     }),
   },
   methods: {
+    unBlur() {
+      if (this.checkedBlur) {
```


```suggestion
      if (this.blurImage) {
```